docs/supported-mutators.md

```diff
@@ -293,32 +295,33 @@ Strings and literals identified to a regex are mutated in the following way:
 
 ## String Literal
 
-| Original                               | Mutated               |
-| -------------------------------------- | --------------------- |
-| `"foo"` (non-empty string)             | `""` (empty string)   |
-| `""` (empty string)                    | `"Stryker was here!"` |
-| `s"foo ${bar}"` (string interpolation) | `s""` ¹               |
+| Mutant operator                 | Original                               | Mutated               |
+| ------------------------------- | -------------------------------------- | --------------------- |
+| FilledStringToEmpty             | `"foo"` (filled string)                | `""` (empty string)   |
+| EmptyStringToFilled             | `""` (empty string)                    | `"Stryker was here!"` |
+| FilledInterpolatedStringToEmpty | `s"foo ${bar}"` (string interpolation) | `s""` ¹               |
+| FilledInterpolatedStringToEmpty | `\`foo ${bar}\``                       | `\`\``                |
```

Comment:
These lines break the website build: https://github.com/stryker-mutator/stryker-mutator.github.io/actions/runs/8295994425/job/22704167287#step:7:39

Reply:
Yep, sorry! https://github.com/stryker-mutator/mutation-testing-elements/pull/3066 fixes that.

Btw, it's not the line breaks, but the way of escaping backticks.